ERCS/erc-7680.md

```diff
@@ -0,0 +1,261 @@
+---
+eip: 7680
+title: Valued Tokens with Layered Liquidity
+description: Layered liquidity protocol to ensure intrinsic value and enhanced stability for tokens
+author: Jackie JC.Lee (@catiga)
+discussions-to: https://ethereum-magicians.org/t/erc2510-embedding-perpetual-value-and-liquidity-in-tokens/19577
```

Comment:
```suggestion
discussions-to: https://ethereum-magicians.org/t/erc-7680-embedding-perpetual-value-and-liquidity-in-tokens/19577
```

Reply:
Yes, I've made erc number changed to correct